### ISSUE TITLE
Broken UI with 2-bytes unicode characters (e.g. 🗲🔋)

### ISSUE THREAD

**Issue by edouard-lopez** [tmux-plugins/tmux-battery#31]
If I set the icons as follow:

```
set -g @batt_charged_icon "✔"
#set -g @batt_charging_icon "🗲"
set -g @batt_discharging_icon '🔋'
set -g @batt_attached_icon '🔌'
```

The UI broke loose when I unplug my laptop and switch to discharging and the tmux status bar appear twice each time it refresh.

![selection_005](https://cloud.githubusercontent.com/assets/1212392/18348455/b28ea464-75cb-11e6-8700-2e44aec8ea68.png)

If I replace charging icon `🗲` (`U+1F5F2 LIGHTNING MOOD`, UTF-16: `0xD83D 0xDDF2`) by `⚡`  (`U+26A1 HIGH VOLTAGE SIGN`, UTF-16: `0x26A1`) there is no more problem while charging.

**Comment by edouard-lopez** [comment]
Temporary solution 
```
set -g @batt_charged_icon '#[fg=green,bold]✔'
set -g @batt_charging_icon '#[fg=yellow,bold]▲'  # ⚡
set -g @batt_discharging_icon '#[fg=red,bold]▼'  # '🔋'
set -g @batt_attached_icon '#[fg=cyan,bold]🔌'  # <-- never seen this one
```

**Comment by martinbeentjes** [comment]
As @adrianluff mentioned in #27, try to update to Vim 8.0 and use the latest version of the plugin. Does that fix this issue?

**Comment by edouard-lopez** [comment]
How is this a vim issue? I tried different unicode characters and all the 2-bytes failed me, so might be terminal or font related.

**Comment by martinbeentjes** [comment]
If you edit your vim configuration file (or the plugin itself) within your terminal and thus added the unicode characters in your terminal, shouldn't bug at that point already?

**Comment by edouard-lopez** [comment]
Indeed, but same if I edit in [xed editor](http://github.com/linuxmint/xed) (with GUI and utf-8 support)

**Comment by martinbeentjes** [comment]
So we can conclude it is a terminal or font related issue?

**Comment by edouard-lopez** [comment]
I'm posting the issue in tmux/tmux#556, we need to find the solution before closing,

**Comment by martinbeentjes** [comment]
If you have any more information about a solution, keep me updated.

**Comment by martinbeentjes** [comment]
As mentioned in tmux/tmux#556, can we conclude that this is an issue which goes deeper than this plugin? If so, I will close this issue.

**Comment by edouard-lopez** [comment]
@martinbeentjes yep, seems it's behind this project's scope
